source/vibe/web/common.d

```diff
@@ -305,8 +321,10 @@ MethodAttribute method(HTTPMethod data)
 	
 	shared static this()
 	{
+		// Tie IAPI.getInfo to "GET /root/foo/info2"
 		registerRestInterface!IAPI(new URLRouter(), new API(), "/root/");
-		// now IAPI.getInfo is tied to "GET /root/info2"
+		// Or just to "GET /foo/info2"
```

Comment:
It is. Comments describe the line under, not above.
So:
- `registerRestInterface!IAPI(new URLRouter(), new API(), "/root/");` => /root/foo/info2
- `registerRestInterface!IAPI(new URLRouter(), new API());` => /foo/info2

But maybe my comments are not clear. I'll rework that.